### ISSUE TITLE
Soccer - Restricting Opponents Positions Error 

### ISSUE THREAD

**Issue by MPSki** [DimaKudosh/pydfs-lineup-optimizer#165]
When I attempt to restrict opponents positions in soccer it does not work and gives a "Game Info isn't specified for players". I am importing the CSV from draft kings. I suspect that this is due to the game info being in a different format than other sports  like baseball which has worked for me.

Baseball format 

NCD@DOO 06/26/2020 05:30AM ET

Soccer format 

BHA vs LIV 03:15PM ET

I could be wrong and wonder if anyone else has encountered this problem before.

**Comment by mbencho** [comment]
Yes, for some reason Draftkings has Soccer formatted differently than everything else. Quick fix is to just convert the Soccer format to the same way as baseball format. I believe the code here keys off of the '@' and that's why it doesn't work. Probably worth sending them an email to try and get it consistent.

**Comment by MPSki** [comment]
Thats what I assumed as well,  I went through and changed the 'vs' to the @ in the game info column but resulted in the same error. A screenshot of an example code and photo of the data in the CSV is below. Sending an email off to them is a good idea when I get some time. 

<img width="1104" alt="Screen Shot 2020-07-08 at 3 43 13 PM" src="https://user-images.githubusercontent.com/67971361/86963106-fcf9a980-c131-11ea-8460-15f28f724786.png">


<img width="889" alt="Screen Shot 2020-07-08 at 3 40 08 PM" src="https://user-images.githubusercontent.com/67971361/86962671-4eedff80-c131-11ea-967d-3d05c2e5c3db.png">

**Comment by Denwen12** [comment]
can you upload the csv cause I can see the problem and want to double check

**Comment by Denwen12** [comment]
importer understands the original csv and you just have wrong info in restrict opposing players

**Comment by Denwen12** [comment]
can you show me your full python code you are running to generate lineups

**Comment by Denwen12** [comment]
should be 
optimizer.restrict_positions_for_opposing_team(['GK'],  
 ['F/UTIL'])

**Comment by MPSki** [comment]
The full code is here and I am still receiving the same error as before.
<img width="1049" alt="Screen Shot 2020-07-09 at 1 14 59 AM" src="https://user-images.githubusercontent.com/67971361/86999964-e418e480-c181-11ea-853b-58fd18e886a3.png">

As for using 'F/UTIL' as a player position it will only take 'F','M','D' and 'GK' as input.
<img width="994" alt="Screen Shot 2020-07-09 at 1 15 50 AM" src="https://user-images.githubusercontent.com/67971361/87000063-28a48000-c182-11ea-997a-bebcbdee8ca6.png">

**Comment by Denwen12** [comment]
I'll try and help in the morning

**Comment by MPSki** [comment]
Hey not sure if you had any time to look into it today but as I have been attempting to solve the issue with no solution yet I have discovered that when importing many soccer CSV that are directly dowloaded from Draft Kings they will not load and throw the following error. Not sure if anyone has noticed this before or know a way to solve it?

<img width="946" alt="Screen Shot 2020-07-10 at 1 29 32 AM" src="https://user-images.githubusercontent.com/67971361/87119942-53a7d600-c24d-11ea-94f4-85078749d1b3.png">

**Comment by mbencho** [comment]
the unicodeDecodeError is due to some of the non-english characters that DK uses on the names, usually ß is the culprit. You need to remove that character from the file. Also, I believe in your original error you need to have the date in the column as well to exactly match the format of the other sports. 

NCD@DOO **06/26/2020** 05:30AM ET

In your example you only have the time, not the date.
